src/services/database/__tests__/message-id-replacement-repository.integration.test.js

```diff
@@ -1,73 +1,88 @@
+import { getAllMessageIdReplacements } from "../message-id-replacement-repository";
+import { createMessageIdReplacements } from "../create-message-id-replacements";
 import { modelName } from '../../../models/message-id-replacement';
-import { getNewMessageIdByOldMessageId } from '../../database/message-id-replacement-repository';
-import { v4 } from 'uuid';
-import { logError } from '../../../middleware/logging';
-import {
-  errorMessages,
-  FragmentMessageIdReplacementRecordNotFoundError
-} from '../../../errors/errors';
-
-jest.mock('../../../middleware/logging');
+import { logError } from "../../../middleware/logging";
+import ModelFactory from '../../../models';
+import { v4 as uuidv4 } from 'uuid';
+import expect from "expect";
 
 describe('message-id-replacement-repository', () => {
-  afterAll(() => {
-    // close the db connection to avoid "Jest did not exit" warning messages
-    MessageIdReplacement.sequelize.close();
+  // Global test variables.
+  let messageIdReplacementRecords;
+  const numberOfRecordsToSeed = 10;
+
+  // Seed the database with test values.
+  beforeAll(async () => {
+    messageIdReplacementRecords = await seedTestData(numberOfRecordsToSeed);
   });
 
-  const uuidv4UpperCase = () => v4().toUpperCase();
-  const MessageIdReplacement = ModelFactory.getByName(modelName);
+  afterAll(async () => {
+    await MessageIdReplacement.truncate();
+    await MessageIdReplacement.sync({ force: true });
+    await MessageIdReplacement.sequelize.close();
+  });
 
-  describe('getNewMessageIdByOldMessageId', () => {
-    it('should return the new message id of a given old message id (in uppercase)', async () => {
-      // given
-      const oldMessageId = uuidv4UpperCase();
-      const newMessageId = uuidv4UpperCase();
-      await MessageIdReplacement.create({
-        oldMessageId,
-        newMessageId
-      });
+  const MessageIdReplacement = ModelFactory.getByName(modelName);
 
+  describe('getAllMessageIdReplacements', () => {
+    it('should retrieve all of the message ids with replacements successfully', async () => {
       // when
-      const newMessageIdFromRecord = await getNewMessageIdByOldMessageId(oldMessageId);
+      const result = await getAllMessageIdReplacements(getAllOldMessageIds(messageIdReplacementRecords));
+
+      const messageIds = {
+        oldMessageIdsFromResult: getAllOldMessageIds(result),
+        oldMessageIds: getAllOldMessageIds(messageIdReplacementRecords),
+        newMessageIdsFromResult: getAllNewMessageIds(result),
+        newMessageIds: getAllNewMessageIds(messageIdReplacementRecords).map(messageId => messageId.toUpperCase())
+      }
 
       // then
-      expect(newMessageIdFromRecord).toBe(newMessageId);
-      expect(newMessageIdFromRecord).toBe(newMessageIdFromRecord.toUpperCase());
+      expect(messageIds["oldMessageIdsFromResult"]).toEqual(messageIds["oldMessageIds"]);
+      expect(messageIds["newMessageIdsFromResult"]).toEqual(messageIds["newMessageIds"]);
```

Comment:
Not necessary, but to be consistent with our other tests, let's change the syntax of these to `messageIds.newMessageIdsFromResult`